docs/domains/KnowledgeArtifactManagement/services/TerminologyService/openapi.yml

```
@@ -0,0 +1,540 @@
+          description: The unique identifier for the ValueSet resource to retrieve.
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/fhir+json:
+              schema:
+                type: object
+                additionalProperties: { }
```

YAMLlint (1.37.1)

[error] too many spaces inside empty braces

(braces)